eng/scripts/build.ps1

```diff
@@ -16,19 +24,33 @@ if (![string]::IsNullOrWhiteSpace($filter)) {
     $keys = $keys.Where( { $_ -match $filter }) 
 }
 
-$keys | ForEach-Object { $sdks[$_] } | ForEach-Object -Parallel {
```

Comment:
Since this is no longer parallel, will this fail fast on the first codegen/build failure, or still run through all of them (might be cumbersome to go through new build/codegen failures run by run as opposed to getting them all at once).

Reply:
It will fail on first one I believe.